src/main/haskell/kore/app/exec/Main.hs

```diff
@@ -60,11 +60,12 @@ import           Kore.Step.Function.Registry
                  ( extractEvaluators )
 import qualified Kore.Step.OrOfExpandedPattern as OrOfExpandedPattern
 import           Kore.Step.Simplification.Data
-                 ( evalSimplifier )
+                 ( evalSimplifierWithTimeout )
 import qualified Kore.Step.Simplification.ExpandedPattern as ExpandedPattern
 import qualified Kore.Step.Simplification.Simplifier as Simplifier
                  ( create )
 import           Kore.Step.Step
+import           Kore.Step.Simplification.Data ( defaultSMTTimeOut )
```

Comment:
Most editors can automatically run stylish-haskell when saving, please use that feature. Otherwise, please run it yourself. If, for whatever reason, you don't want to do that, please do not duplicate imports and please order imports alphabetically within their group.